custom_components/audiconnect/dashboard.py

```diff
@@ -390,178 +393,302 @@ def create_instruments():
         TripData(attr="longterm_reset", name="LongTerm Trip User Reset"),
         Lock(),
         Preheater(),
-        Sensor(attr="model", name="Model", icon="mdi:car-info", unit=None),
-        Sensor(attr="mileage", name="Mileage", icon="mdi:speedometer", unit="km"),
+        Sensor(
+            attr="model",
+            name="Model",
+            icon="mdi:car-info",
+            unit=None,
+            state_class=None,
+            device_class=None,
+        ),
+        Sensor(
+            attr="mileage",
+            name="Mileage",
+            icon="mdi:speedometer",
+            unit="km",
+            state_class="total_increasing",
+            device_class=None,
+        ),
         Sensor(
             attr="service_adblue_distance",
             name="AdBlue range",
             icon="mdi:gas-station",
             unit="km",
+            state_class=None,
+            device_class=None,
+        ),
+        Sensor(
+            attr="range",
+            name="Range",
+            icon="mdi:gas-station",
+            unit="km",
+            state_class=None,
+            device_class=None,
         ),
-        Sensor(attr="range", name="Range", icon="mdi:gas-station", unit="km"),
         Sensor(
             attr="hybrid_range",
             name="hybrid Range",
             icon="mdi:gas-station-outline",
             unit="km",
+            state_class=None,
+            device_class=None,
         ),
         Sensor(
             attr="service_inspection_time",
             name="Service inspection time",
             icon="mdi:room-service-outline",
             unit="days",
+            state_class=None,
+            device_class=None,
         ),
         Sensor(
             attr="service_inspection_distance",
             name="Service inspection distance",
             icon="mdi:room-service-outline",
             unit="km",
+            state_class=None,
+            device_class=None,
         ),
         Sensor(
-            attr="oil_change_time", name="Oil change time", icon="mdi:oil", unit="days"
+            attr="oil_change_time",
+            name="Oil change time",
+            icon="mdi:oil",
+            unit="days",
+            state_class=None,
+            device_class=None,
         ),
         Sensor(
             attr="oil_change_distance",
             name="Oil change distance",
             icon="mdi:oil",
             unit="km",
+            state_class=None,
+            device_class=None,
+        ),
+        Sensor(
+            attr="oil_level",
+            name="Oil level",
+            icon="mdi:oil",
+            unit="%",

```

Comment:
Unit should be imported from HA as well.

Reply:
on it